src/AssetPublisher.php

```diff
@@ -244,7 +245,9 @@ private function hash(string $path): string
             return ($this->hashCallback)($path);
         }
 
-        $path = (is_file($path) ? dirname($path) : $path) . FileHelper::lastModifiedTime($path);
+        $dirname = is_file($path) ? dirname($path) : $path;
```

Comment:
Path can be file? Isn't it always a directory?

Reply:
> Path can be file? Isn't it always a directory?

I save this part from current version

Reply:
OK